pkg/clients/httpRateLimiter.go

```diff
@@ -0,0 +1,64 @@
+//
+// Copyright 2024 The GUAC Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package clients
+
+import (
+	"github.com/guacsec/guac/pkg/logging"
+	"github.com/guacsec/guac/pkg/version"
+	"golang.org/x/time/rate"
+	"net/http"
+	"time"
+)
+
+var (
+	osvDevLimiter         = rate.NewLimiter(rate.Every(time.Minute), 10000)
+	clearlyDefinedLimiter = rate.NewLimiter(rate.Every(time.Minute), 2000)
```

Comment:
I like the pattern in the grpc limiter better where the client code passes in the limiter. This package shouldn't have the osv/cd limiters defined.

Reply:
Thanks, the osv/cd files are now calling `NewRateLimitedTransport` with their own rate limits.